samples/matter/common/cmake/source_common.cmake

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+set(MATTER_COMMONS_SRC_DIR ${ZEPHYR_NRF_MODULE_DIR}/samples/matter/common/src)
+
+target_include_directories(app PRIVATE
+    ${MATTER_COMMONS_SRC_DIR}
+    ${NLIO_ROOT}/include
```

Comment:
These two: NLIO and app-common are probably not needed anymore, right? They're pulled by sdk-connectedhomeip CMake modules.